galley/docker/Dockerfile.galley

```diff
@@ -6,7 +6,8 @@ ARG BASE_VERSION=latest
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
 FROM docker.io/istio/base:${BASE_VERSION} as default
-USER istio-proxy
+
+USER 1337:1337
```

Comment:
does this not work between line 18 and 19?  I recognize distroless has no /etc/paswd entry with 1337, however, I would be surprised if the application actually cared.

Reply:
Why are we running galley as 1337? I thought 1337 was reserved for the proxy?

Reply:
`nobody` / `65534 ` should be a better choice since this user is included in distroless (e.g. https://github.com/GoogleContainerTools/distroless/pull/335) and galley shouldn't need any sudo rights?

Reply:
I see. Do we have problems when we have control plane and proxy alongside each other (like we do for Pilot)? I would assume no because they are in different containers?

Reply:
Yes. You are right. There is no problems because they are in different containers.